src/config.rs

```diff
@@ -669,13 +679,25 @@ pub struct PrimaryColors {
     foreground: Rgb,
 }
 
+#[derive(Debug, Deserialize)]
+pub struct CursorColors {
```

Comment:
I changed it to use PrimaryColors, but I split it out in case PrimaryColors gets new elements in the future. No matter, whoever does that in the future can split it back out.